hack/eks/install.sh

```diff
@@ -75,7 +75,12 @@ capact::aws::install::efs_csi_driver() {
 
 capact::aws::install::capact() {
   shout "Deploying Capact..."
-  "${CURRENT_DIR}"/cluster-components-install-upgrade.sh
+  capact install --environment eks \
```

Comment:
Currently, `--environment eks` is not used, and it's quite inconsistent with ` --environment kind`.

Which approach we choose?
1. Set overrides via flags
2. Embed overrides in Capact CLI
3. Rename the `--environment` to sth kind related setup? BTW it's even not too much `kind` related setup but more `local` setup. With k3d we will apply the same overrides.

What I like about the current approach is that I can download and execute `capact install --env kind` and that's it. I do not have to download/set any overrides. On the other hand, embed overrides for eks, gke etc. may be hard to maintain/use.

@lukaszo @pkosiec any thoughts?

_NOTE:_
After final decision, get rid of `TODO` comments in Capact CLI: https://github.com/capactio/capact/search?l=Go&q=eks

Reply:
I like 3rd option the most. The second most preferred solution by me would be the No 2 one. However, I think the AWS-specific overrides are opinionated (e.g. whether the load balancer should be internal or not), so I think we don't need dedicated ones. So, we don't need `--environment aws` at all.

Reply:
From what I see `--environment eks` is not used anywhere in CLI and only passed to not apply `kind` vars. Am I right?

In that case maybe we can just pass `--environment none` and pass all in flags?

Reply:
@lukaszo 
> is not used anywhere in CLI
yes, because instead of embedding, overrides are passed via flags. 

>In that case maybe we can just pass --environment none and pass all in flags?
This could work, but still we should change the `kind` env as overrides are not kind related. Additionally, we need to get rid of `TODOs` comments, plus probably get rid of `--env` flag at all as it may be misleading, we have currently `kind`, in `TODOs` we put support for `eks`, but now we have situation when for `kind` we have embedded overrides and enable them via `--env=kind` and for EKS, GKE, we set `--env=eks/gke` which skips the local related overrides and other overrides are passed via flags. For me, this inconsistency is bad, mostly because of UX.